packages/components/webpack.config.ts

```diff
@@ -40,6 +40,7 @@ const config: Configuration = {
               ],
               '@babel/preset-typescript',
             ],
+            plugins: [['@babel/plugin-transform-modules-commonjs', { loose: true }]],
```

Comment:
why?

Reply:
For mocking hooks this is needed. For testing, being able to mock hooks is really important.

Reply:
As aligned, we decided to keep this plugin